src/main/kotlin/minesweeper/controller/InputFilter.kt

```diff
@@ -11,7 +11,7 @@ object InputFilter {
             val position = InputParser.parsePosition(InputView.inputNumber())
             Position(position)
         } catch (e: Exception) {
-            InputView.printError(e.message!!)
+            e.message?.let { InputView.printError(it) }
```

Comment:
이러한 구문을 함수 reference로 넘길 수 있습니다.
```suggestion
            e.message?.let(InputView::printError)
```
lambda로 작성하는 것과 어떤 차이가 있는지 참고할 수 있는 글을 소개해드릴게요 :slightly_smiling_face:
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845
참고로 저는 가능한 곳에서는 꼭 함수 reference를 활용하는 것을 지향합니다. lambda로 가져온 데이터를 별도로 조작하지 않고 그대로 함수 시그니처로 넘긴다는 것을 드러낼 수 있기 때문입니다.

Reply:
조언해 주신대로, 이런 상황에서는 함수 reference를 활용하는 것이 더욱 직관적인것 같습니다 😀

- https://github.com/next-step/kotlin-minesweeper/commit/ecf7d91de7181d775f8b2175c54f9a81d750aa28